pom.xml

```
@@ -134,7 +134,7 @@
       <dependency>
         <groupId>com.fasterxml.jackson.core</groupId>
         <artifactId>jackson-databind</artifactId>
-        <version>2.10.0.pr1</version>
+        <version>2.12.6.1</version>
```

```suggestion
        <version>2.13.2.20220324</version>
```
to align with spotify-bom